### PR TITLE
added echo to builtins

### LINKED ISSUE #14: Echo

### PR DESCRIPTION
**Pull request by danReynolds**
- [x] Implement basic echo, printing to console
- [x] Add -n flag for supressing newline

closes #14

### CODE REVIEW

**Comment by danReynolds** [comment]
Without ok() it won't return an result i32 though.

**Comment by nikklassen** [comment]
You're not returning the value of that statement

**Comment by danReynolds** [comment]
thought you meant the ok at the end, for some reason i couldn't see your comment in the code diff.

File: src/builtin.rs [hunk 1]
```diff
@@ -27,6 +30,30 @@ pub fn exit(_args : &[&str]) -> io::Result<i32> {
     process::exit(0)
 }
 
+pub fn echo(args : &[&str]) -> io::Result<i32> {
+    let mut opts = Options::new();
+
+    opts.optflag("n", "", "Suppress new lines");
+
+    let matches = match opts.parse(&args[..]) {
```
Comment (nikklassen) [inline src/builtin.rs]: I'd prefer this to be a try.  Also you can change ```&args[..]``` to ```args``` since it's already a &[].

Comment (danReynolds) [inline src/builtin.rs]: it breaks if you do args

**Pull request merged**